src/index.ts

```diff
@@ -71,6 +71,19 @@ async function getRegionsInteractive(message: string) {
     return region;
 }
 
+async function deletePostsInteractive(job: Job, jobInfo: JobInfo, regionNames: string[], similar: number) {
+    const prompt = new Toggle({
+        message: "Do you want to delete job posts?",
```

Comment:
Just a small suggestion
```suggestion
        message: "Do you want to remove existing job posts?",
```

Reply:
Can we update to have Yes by default, since we know that the common use case is to delete all previous posts?